tests/smoke/coc_citizen.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+SMOKE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+. ${SMOKE_DIR}/base.sh code_of_conduct="Citizen Code Of Conduct (Suitable for large communities and events)"
+. ${SMOKE_DIR}/base.sh code_of_conduct="Contributor Covenant (Recommended for projects of all sizes)"
+. ${SMOKE_DIR}/base.sh code_of_conduct="None"
```

Comment:
please keep an empty line at the end of the file.

probably you need to install pre-commit in your environment, so it will do it automatically before every git commit command:

```bash
$ pre-commit install
```

ref: https://github.com/osl-incubator/cookiecutter-python/blob/main/.pre-commit-config.yaml#L5